client/src/App.js

```diff
@@ -15,7 +15,10 @@ function App() {
     fetch(`http://127.0.0.1:3001/website?url=${url}&width=${width}&height=${height}`)
       .then(async response => {
         if (response.ok) {
-          let json = await response.json();
+          const json = await response.json();
+          if (json.data.url) {
+            json.data.url = `http://localhost:3000/redirect/${json.data.url}`;
+          }
```

Comment:
I think it'd be good to add a clarifying comment of why this is necessary.

I imagine it has to do with the proxying that's happening in `setupProxy.js`?